OpenBCI_32bit_Library.h

```diff
@@ -105,6 +105,9 @@ class OpenBCI_32bit_Library {
   void    accelWriteAxisDataSerial(void);
   void    activateAllChannelsToTestCondition(byte testInputCode, byte amplitudeCode, byte freqCode);
   void    activateChannel(byte);                  // enable the selected channel
+  void    activateLedMarker(boolean);
+  void    activateLedSDWrite(boolean);
```

Comment:
Why do SD in the function name? That's confusing me.

Reply:
It should be read as "activate the LED sequence to show SD card writes are happening".  Changed it to:
+  void    activateLedMarkerRec(boolean);
+  void    activateLedSDCardWrite(boolean);